src/commands/reset.ts

```diff
@@ -34,9 +36,15 @@ export async function _reset(
       }
       const databaseOwner = parsedSettings.databaseOwner;
       const logSuffix = shadow ? "[shadow]" : "";
-      await pgClient.query(
-        `DROP DATABASE IF EXISTS ${escapeIdentifier(databaseName)};`,
-      );
+      if (force) {
+        await pgClient.query(
+          `DROP DATABASE IF EXISTS ${escapeIdentifier(databaseName)} WITH (FORCE);`,
```

Comment:
Introduced in PG13, so this is technically supported by all supported versions of PostgreSQL now. Interesting.